tasks/lib/photobox.js

```diff
@@ -505,10 +505,15 @@ PhotoBox.prototype.startPhotoSession = function() {
   this.pictures.forEach( function( picture ) {
     this.grunt.log.writeln( 'started photo session for ' + picture );
 
+    var parsedImage = require( 'url' ).parse(picture);
```

Comment:
I cant use `require( 'url' )` inside of child process, so adding it here to pass image name together with url :(